llvm/test/tools/llvm-readobj/COFF/cetcompat.test

```diff
@@ -1,7 +1,6 @@
-# To regenerate has-cet.exe
-# $ echo int main() { return 0; } > has-cet.c
-# $ cl has-cet.c /link /cetcompat
-RUN: llvm-readobj --coff-debug-directory %p/Inputs/has-cet.exe | FileCheck %s
+RUN: yaml2obj %p/Inputs/hascetret42.yaml -o %t.obj
+RUN: lld-link /out:%t.exe /entry:main /cetcompat %t.obj
```

Comment:
You can't run `lld` in the `llvm` testsuite - that's a layering violation; the `lld` subproject might not be enabled/available while running these tests.